### ISSUE TITLE
Define Keycloak versions since the first version of the repo (25)

### ISSUE THREAD

**Issue by djowalker** [slemke/keycloak-testcontainer#173]
Hello, i am wondering if the following could be done, i saw 
```
Currently, this package only supports the latest version of Keycloak. Older releases will work for older versions, but don't expect new releases to work for older Keycloak versions.
```

However in the constructor of `container.ts` here is what could be done :

```
	constructor(version: string="latest") {
		super(`quay.io/keycloak/keycloak:${version}`,);
```
Or maybe allow only some versions with defined version in an enum.
So when using the instance, we could define what version we want to use.

I just saw that it was done in this commit : https://github.com/slemke/keycloak-testcontainer/commit/b2c4e67d1826ba13324cb903508a50475a251015

My personal use is about using just a little older version like the 25.

**Comment by slemke** [comment]
Hey @djowalker, you are right, earlier versions of this package had support for different Keycloak versions. The change to latest was mostly a support thing. While it is easy to input a version in the constructor, I wanted to keep the amount of support low until a feature requests for "more version support" comes along.

If I would implement older versions, I would probably add a configuration object like so:

```js
new KeycloakContainer({
  version: '26.0.7',
  repository: 'quay.io/keycloak/keycloak'
});
```

Both attributes would be optional and fall back to the latest Keycloak version (`quay.io/keycloak/keycloak:latest`). This way everyone could use the version they want (even custom ones), but support would be limited the last two major versions from Keycloak: https://quay.io/repository/keycloak/keycloak?tab=tags

Would that help you out?

**Comment by djowalker** [comment]
@slemke Yep it looks like it is what i need :)

**Comment by slemke** [comment]
@djowalker I released a new version (0.0.8) that allows you to do this:

```js
import KeycloakContainer from 'keycloak-testcontainer';

const container = await new KeycloakContainer({ tag: '25.0' })
    .start();
```

You can finde more info [here](https://github.com/slemke/keycloak-testcontainer?tab=readme-ov-file#starting-a-container-with-a-custom-version). Maybe you can give it a go and let me know if it works for you / you have an issues?

**Comment by djowalker** [comment]
Seems good for this purpose thank you for your fast answer.
